### PR TITLE
[Selenium] Stabilize happy-path test, add  devworkspace-che logs to test report

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
Stabilize happy-path test, add `devworkspace-che` logs to test report, use latest `che-operator` templates.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/19643

### CODE REVIEW

File: .ci/common.sh [hunk 1]
```diff
@@ -20,6 +20,8 @@ function bumpPodsInfo() {
     TARGET_DIR="${ARTIFACT_DIR}/${NS}-info"
     mkdir -p "$TARGET_DIR"
 
+    oc get pods -n ${NS}
```
Comment (sleshchenko) [inline .ci/common.sh:23]: Is it really useful? Maybe then at least separate it from everything else with something like
```
echo "===== Bumpings pods logs in namespace $NS. Pods: ======"
oc get pods -n ${NS}
```

Comment (SkorikSergey) [reply .ci/common.sh:23]: It helps to quickly find pods status when build fail.
For example, part of `build-log.txt` from last build:
```
+ TARGET_DIR=/logs/artifacts/devworkspace-che-info
+ mkdir -p /logs/artifacts/devworkspace-che-info
+ oc get pods -n devworkspace-che
NAME                                       READY   STATUS             RESTARTS   AGE
devworkspace-che-manager-768b7dd9c-gkj57   0/1     CrashLoopBackOff   8          19m
```

**Review by sleshchenko — APPROVED** [review]

**Pull request merged**